conda_env_tracker/deploy.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+from __future__ import print_function
+
+import datetime
+from glob import glob
+import os
+import time
+
+from git import Repo
+import conda.api
+import conda.fetch
+from conda_env_tracker.cli import tempdir, create_tracking_branches
+from conda_execute.lock import Locked
+
+manifest_branch_prefix = 'manifest/'
+
+
+def tags_by_label(repo, env_branch):
+    env_branch.checkout()
+    label_dir = os.path.join(repo.working_dir, 'labels')
+    tags = {}
+    if os.path.isdir(label_dir):
+        for label_fname in glob(os.path.join(label_dir, '*.txt')):
+            with open(label_fname, 'r') as fh:
+                tag = fh.read()
+            label = os.path.splitext(os.path.basename(label_fname))[0]
+            tags[label] = tag
+    return tags
+
+
+def deploy_tag(repo, tag_name, target):
+    tag = repo.tags[tag_name]
+    # Checkout the tag in a detached head form.
+    repo.head.reference = tag.commit
+    repo.head.reset(working_tree=True)
+
+    # Pull out the environment name from the form "env_<env_name>_2000_12_25".
+    env_name = tag_name.split('-')[1]
+    deployed_name = tag_name.split('-', 2)[2]
```

Comment:
More seriously, your comment uses an underscore as the separator but the code uses a minus.